cmd/identities/import.go

```diff
@@ -20,8 +20,16 @@ import (
 // importCmd represents the import command
 var importCmd = &cobra.Command{
 	Use:   "import <file.json [file-2.json [file-3.json] ...]>",
-	Short: "import identities from files or STD_IN",
-	Long:  "Import identities from files or STD_IN. Files are expected to each contain a single identity. The validity of files can be tested beforehand using `... identities validate`. Importing credentials is not yet supported.",
+	Short: "Import identities from files or STD_IN",
+	Long: `Import identities from files or STD_IN:
+
+	kratos identities import file.json
+
+	kratos identities import <(cat file.json)
```

Comment:
JSON via argument is not supported.
```suggestion
	cat file.json | kratos identities import
```